src/main/resources/schema.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE USER(
+  USER_ID BIGINT AUTO_INCREMENT PRIMARY KEY,
+  FIRST_NAME VARCHAR(30) NOT NULL,
```

Comment:
Zašto radiš svoj ddl umjesto korištenja hibernate? Ne kažem da li je to dobra ili loša odluka samo hoću čuti tvoj razlog za odabir ovog pristupa.

Reply:
To sam stavio koliko sam gledao po tutorialima, da kreiram tako lokalno bazu da mi se ne isprazni svaki put kad loadam project, i dodao sam u application.properties spring.jpa.hibernate.ddl-auto=none. Mogu ja promijeniti da se koristi hibernate.

Reply:
Postoji razlika između embeddane i in-memory baze podataka. Embeddana će sačuvati sve podatke, samo će ih spremiti u jednostavan file za razliku od nekog komercijalnog RDMS-a. Također ako hibernate vidi da već imaš potrebne tablice, on ih neće prebrisati ali hoće dodati stvari poput novih kolumna.